### PR TITLE
Replaces the particular boolean returns with a boolean expression

### LINKED ISSUE #1: Avoid the "afraid of boolean"
Evaluate the boolean expression and return the result instead of using multiple if / return False / return True on the is_winner method from the Player class.

### PR DESCRIPTION
**Pull request by apojomovsky**
Avoids the particular return cases on the is_winner method by returning the evaluation of a boolean expression. Fixes #1

### CODE REVIEW

File: game/player.py [hunk 1]
```diff
@@ -12,9 +12,7 @@ def color(self):
         return self._color
 
     def is_winner(self, board):
-        if board.game_over() and board.winner_color() == self._color:
-            return True
-        return False
+        return board.game_over() and board.winner_color() == self._color
```
Comment (basicNew) [inline game/player.py]: Just a very minor knit, but in general it is a good practice to append an `is_` when defining inquiring messages that return boolean values. So in this case `board.is_game_over()` is more explicit and leaves no doubts about what it does (on the other hand, `board.game_over()` in other context may mean that you are actually finishing the game)

**Review by basicNew — COMMENTED** [review]
Minor comment

**Comment by apojomovsky** [comment]
Updated PR with:
- Changes in method name
- Applied the boolean approach to another function return

**Review by basicNew — COMMENTED** [review]
Great, LGTM!

**Pull request merged**